rules/no-single-promise-in-promise-methods.js

```diff
@@ -0,0 +1,97 @@
+'use strict';
+const {isMethodCall} = require('./ast/index.js');
+const needsSemicolon = require('./utils/needs-semicolon.js');
+
+const MESSAGE_ID_ERROR = 'no-single-promise-in-promise-methods/error';
+const MESSAGE_ID_SUGGESTION_1 = 'no-single-promise-in-promise-methods/suggestion-1';
+const MESSAGE_ID_SUGGESTION_2 = 'no-single-promise-in-promise-methods/suggestion-2';
+const messages = {
+	[MESSAGE_ID_ERROR]: 'Parameter in `Promise.{{method}}()` should not be a single element array.',
```

Comment:
How about, ``Wrapping a single element array with `Promise.{{method}}()` is unnecessary.``

Reply:
It's done.